test/config/TestConfig.cpp

```diff
@@ -1456,7 +1467,9 @@ TEST_F(ConfigTestFixture, SensorPublishDisableFeature)
     PlainConfig config;
     config.LoadFromJson(jsonView);
 
+#if !defined(EXCLUDE_SENSOR_PUBLISH)
```

Comment:
I was thinking of excluding Sensor publish tests completely instead of only excluding the assert statements.